### PR TITLE
Web: support base global type and fix the remaining renderers

### PR DESCRIPTION
**Pull request by abock**

### CODE REVIEW

File: Agents/Xamarin.Interactive.Console/ConsoleAgent.cs [hunk 1]
```diff
@@ -46,6 +46,9 @@ internal DotNetFrameworkEvaluationContextManager (ConsoleAgent agent)
             {
             }
 
+            protected override object CreateGlobalState ()
+                => new EvaluationContextGlobalObject ((DotNetCoreAgent)Context);
```
[the remark this answers is not in the document]
Comment (sandyarmstrong) [reply Agents/Xamarin.Interactive.Console/ConsoleAgent.cs]: Looks like the build notice this one too. 👍 otherwise on this beautiful PR.

**Comment by sandyarmstrong** [comment]
🔖

**Comment by abock** [comment]
It's the bots, man.

**Comment by sandyarmstrong** [comment]
I meant it in the metaphorical sense.

**Comment by abock** [comment]
Still probably the bots though.

**Pull request merged**